setup.cfg

```diff
@@ -14,10 +14,6 @@ replace = __version__ = '{new_version}'
 [bdist_wheel]
 universal = 1
 
-[aliases]
-# Define setup.py command aliases here
-test = pytest
-
 [tool:pytest]
-collect_ignore = ['setup.py']
```

Comment:
This was warning about an unknown option: https://github.com/terencehonles/pyuwsgimemhog/runs/2421912545?check_suite_focus=true#step:6:36 and the change to use `addopts` is how I've used flags in the past.